gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -377,6 +377,21 @@
 		<type>\$type.type</type>
 		<nports>\$nchan</nports>
 	</$sourk>
+#if $sourk == 'source'
+	<source>
+		<name>rx_metad</name>
```

Comment:
I actually had `rx_metadata` there first :) but it rendered the block virtually unusable in GRC - all sink pads then expand to a length that really feels uncomfortable for usage, since it was longer than the typical inter-block spacing, even for flowgraphs < 0.2µBalint.